yt_dlp/extractor/lrt.py

```diff
@@ -2,16 +2,61 @@
 from ..utils import (
     clean_html,
     merge_dicts,
+    try_get
 )
 
 
-class LRTIE(InfoExtractor):
-    IE_NAME = 'lrt.lt'
+class LRTStreamIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?lrt\.lt/mediateka/tiesiogiai/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://www.lrt.lt/mediateka/tiesiogiai/lrt-opus',
+        'info_dict': {
+            'id': 'lrt-opus',
+            'live_status': 'is_live',
+            'title': 're:^LRT Opus.+$',
+            'ext': 'mp4'
+        }
+    }]
+
+    def _extract_js_var(self, webpage, var_name, default):
+        return self._search_regex(
+            r'%s\s*=\s*(["\'])((?:(?!\1).)+)\1' % var_name,
+            webpage, var_name.replace('_', ' '), default, group=2)
```

Comment:
this code is repeated. Either move it into a base class, or, if the code is robust enough*, this could be useful in common.py as well

\* it needs a `\s`/`\b` at the start. Otherwise, looks robust to me

Reply:
I'd prefer to keep it in this file for now as I think it could be extended further e.g. there could be an optional `var` before the variable's name. So, we could open this can of worms in the future IMHO